plugins/feeds/public/cisco_umbrella_top_domains.py

```diff
@@ -0,0 +1,39 @@
+import io
+from datetime import timedelta
+from typing import ClassVar
+
+from core import taskmanager
+from core.config.config import yeti_config
+from core.schemas import task
+from core.schemas.observables import hostname
+
+
+class CiscoUmbrellaTopDomains(task.FeedTask):
+    _defaults = {
+        "frequency": timedelta(hours=1),
+        "name": "CloudflareTopDomains",
+        "description": "Import Cloudflare top domains",
+    }
+    _SOURCE: ClassVar[
+        "str"
+    ] = "http://s3-us-west-1.amazonaws.com/umbrella-static/top-1m.csv.zip"
+
+    def run(self):
+        top_domains = int(yeti_config.get("umbrella", "top_domains", 10000))
+        response = self._make_request(self._SOURCE, sort=False)
+        data = self._unzip_content(response.content)
+        context = {
+            "name": self.name,
+        }
+        feed = io.BytesIO(data)
+        top_domains = int(top_domains)
```

Comment:
```suggestion
        top_domains = top_domains
```

It's already an int

Reply:
Fixed in e5e40e9b30016c5ac14f85450cc9862ebf2b0095